pandas/tests/indexing/test_interval_new.py

```diff
@@ -0,0 +1,250 @@
+import pytest
+import numpy as np
+import pandas as pd
+
+from pandas import Series, IntervalIndex, Interval
+import pandas.util.testing as tm
+
```

Comment:
same deal here, skip the entire module for now and remove the xfails on individual tests.